swift/Workflow/Sources/Sink.swift

```diff
@@ -1,4 +1,5 @@
-/// Sink is a type that receives incoming values (commonly events or actions)
+/// Sink is a type that receives incoming values (commonly events or `WorkflowAction`)
+/// Use `RenderContext.makeSink` to create instances.
```

Comment:
Nit: I think it is conventional in swift docs to put a blank line between these.

Reply:
Do you mean btw the comment and the code? Not seeing that.

Reply:
Summary and description (just like java/kdoc)